src/Console/TestSuite/StubConsoleOutput.php

```diff
@@ -40,6 +40,19 @@ class StubConsoleOutput extends ConsoleOutput
      */
     protected array $_out = [];
 
+    /**
+     * Constructor
+     *
+     * Closes and unsets the file handle created in the parent constructor to
+     * prevent 'too many open files' errors.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        fclose($this->_output);
+        unset($this->_output);
```

Comment:
I know you've fixed this problem before. I thought we added a destruct somewhere to do this.

Reply:
We fixed it for StubConsoleInput. These stub class issues point to missing interfaces though. I might add interfaces for ConsoleInput and ConsoleOutput so that stubs don't have to inherit from the concrete classes.